packages/app-mobile/components/voiceTyping/AudioRecordingBanner.tsx

```
@@ -94,30 +93,28 @@ const recordingToSaveData = async (recording: Audio.Recording) => {
-		playsInSilentModeIOS: false,
-		staysActiveInBackground: false,
-	});
-};
 
-const useAudioRecorder = (onFileSaved: OnFileSavedCallback, onDismiss: ()=> void) => {
-	const [permissionResponse, requestPermissions] = Audio.usePermissions();
+
+const useJoplinAudioRecorder = (onFileSaved: OnFileSavedCallback, onDismiss: ()=> void) => {
+	const [permissionResponse, setPermissionResponse] = useState(null);
+	const recording = useExpoAudioRecorder(RecordingPresets.HIGH_QUALITY);
+const requestPermissions = useCallback(async () => {
```

ESLint

[error] Expected indentation of 1 tab but found 0.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 0.

(indent)